s3/cmd/main_test.go

```diff
@@ -0,0 +1,31 @@
+package main
```

Comment:
Didn't get the purpose of this

Reply:
yes, this is the way to run the server, to get coverage for any kind of FT or automation testing

Reply:
Same as above